pkg/network/dhcp/dhcp.go

```diff
@@ -43,22 +44,31 @@ func (p *ProbeOutput) IPNet() (*net.IPNet, error) {
 func Probe(ctx context.Context, inf string) (output ProbeOutput, err error) {
 	// use udhcpc to prope the interface.
```

Comment:
probe

Reply:
consider to add `misspell` to the golangci-lint config https://golangci-lint.run/usage/linters/#misspell